src/component/documentation/async-api.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Link } from 'gatsby';
+
+import AsyncApiComponent from './AsyncApiComponent';
+import Layout from '../Layout';
+
+const styles = {
+  container: {
+    display: 'flex',
+  },
+  listGroup: {
+    margin: '60px 20px 60px 0',
+  },
+  subtitle: {
+    fontSize: '.8em',
+    color: '#999',
+  },
+  asyncapi: {
+    position: 'relative',
+    flex: 1,
+  },
+};
+
+const getServiceName = (raw) => {
+  const url = new URL(raw);
+  const path = url.pathname.split('/');
+  return path[2];
+}
+
+const Page = ({ pageContext: { moduleName, module, modules, auth_url } }) => (
+  <Layout pageTitle={`Console - ${module.title}`} breadcrumbs={[{ link: '/install', label: 'Install', active: true }]} className="body-green">
+    <section id="console" className="console section">
+      <div className="container" style={styles.container}>
+        <div className="list-group" style={styles.listGroup}>
+          {Object.keys(modules).map(m => {
+
+            return modules[m].apiEvents && <Link key={m} to={`/documentation/events/${m}`} className={`list-group-item list-group-item-action ${m === moduleName ? 'disabled' : ''}`}>
+              {modules[m].title}
+              <div style={styles.subtitle}>{modules[m].repository}</div>
+            </Link>
+          })}
+        </div>
```

Comment:
Open suggestion, to be like API Reference. 
- Remove sub navigation
- Enable sidebar from async (could help to navigate quickly thought all events)

Reply:
Disucssed. We will improve this point from in another work.